docs/build/program/run.md

```diff
@@ -77,62 +77,53 @@ If you subsequently lose internet connectivity, but stay connected to LAN or WAN
 
 ## Run code on-machine
 
-If you run [PID control loops](https://en.wikipedia.org/wiki/PID_controller) or your machines have intermittent or no network connectivity, you can ensure this does not interfere with the code's execution by running the control code on the same board that is running `viam-server`.
+You can run SDK code directly on your machine.
+If you run [PID control loops](https://en.wikipedia.org/wiki/PID_controller) or your machines have intermittent or no network connectivity, you can ensure lags in communication do not interfere with the machine's performance by running the control code on the same board that is running `viam-server`.
+Running everything on one machine is also convenient if you have a machine (for example, an air quality sensor) that runs all the time, and you don't want to have to connect to it from a separate computer constantly.
 
-{{<imgproc src="/build/program/on-robot.png" resize="800x" declaredimensions=true alt="A client running on a machine">}}
```

Comment:
why remove this?

Reply:
See above

Reply:
The arrow and the box surrounding the client is different. It's subtle but I think it would be better to make that less subtle than to delete

Reply:
Woowwww that is so subtle haha. I totally played "spot the difference" before I first took this out and didn't notice--totally interpreted the big grey box as just a background. Okay will revert this and made a BD ticket and docs ticket to make less subtle.

Reply:
^^Done